### PR TITLE
Fixed shadows  on simulation data

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: plugins/engines/ospray/ispc/render/SimulationRenderer.ispc [hunk 1]
```diff
@@ -780,6 +784,35 @@ inline void computeIndirectShading(DifferentialGeometry& dg,
...
 
+inline void processShadows(DifferentialGeometry& dg,
+                           varying ScreenSample& sample, const varying Ray& ray,
+                           ShadingAttributes& attributes)
+{
+    for (uniform int i = 0; i < attributes.self->abstract.numLights; ++i)
+    {
+        const uniform Light* uniform light =
+            attributes.self->abstract.lights[i];
+        const vec2f s = make_vec2f(0.5f);
```
Comment (tribal-tec) [inline plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: what's the purpose of this value? comment or better name.

File: plugins/engines/ospray/ispc/render/SimulationRenderer.ispc [hunk 2]
```diff
@@ -780,6 +784,35 @@ inline void computeIndirectShading(DifferentialGeometry& dg,
...
+        const bool shadowsEnabled = attributes.lightEmissionIntensity <
+                                    attributes.self->samplingThreshold;
+        attributes.shadowIntensity +=
+            shadowsEnabled ? shadedLightIntensity(sample, ray, attributes,
+                                                  lightDirection, dg)
+                           : 0.f;
+    }
+
+    if (attributes.shadowIntensity > 0.f)
+        // Remove specular color is surface is in the shades
```
Comment (tribal-tec) [inline plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: 'is surface' -> 'if surface'

File: plugins/engines/ospray/ispc/render/SimulationRenderer.ispc (hunk 1 above)
Comment (favreau) [reply plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: replaced by samplingLocation

File: plugins/engines/ospray/ispc/render/SimulationRenderer.ispc (hunk 2 above)
Comment (favreau) [reply plugins/engines/ospray/ispc/render/SimulationRenderer.ispc]: Done

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Pull request merged**